src/Microsoft.Health.Dicom.Operations.Functions.Client/Configs/FunctionsClientConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Health.Dicom.Operations.Functions.Client.Configs
+{
+    public class FunctionsClientConfiguration
+    {
+        public const string SectionName = "Functions";
+
+        [Required]
+        public Uri BaseAddress { get; set; }
+
+        [Required]
+        public OperationRoutesConfiguration Routes { get; set; } = new OperationRoutesConfiguration();
+
+        [Range(0, int.MaxValue)]
+        public int MaxRetries { get; set; }
```

Comment:
nit:Do we want to give default value incase it's missed from configuration?

Reply:
As of now, I have no retries by default lol. I suppose there can be a more generous default!